docs/debugging_tips.md

```diff
@@ -35,19 +35,135 @@ configuration, diagnostic, and troubleshooting data to be provided to Red Hat Te
 
 > See [Generating sos reports for technical support](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/generating_sos_reports_for_technical_support/index) for more information on the `sos` utility usage.
 
-Log into the host running MicroShift and execute the following command to generate an obfuscated
-report that should not contain sensitive information.
+> See [documentation](https://github.com/sosreport/sos) for further information on `sos` utility.
+
+Sos tool is composed of different plugins for tailored retrieval of information from different applications.
+A MicroShift specific plugin has been added from sos version 4.5.1, being able to gather:
+* MicroShift configuration and version
+* YAML output for cluster-wide and system namespaced resources
+* OVNK information
+
+Plugin is automatically enabled as soon as it detects any of the following conditions:
+* MicroShift RPMs being installed.
+* MicroShift systemd service being present, whether running or not.
+* MicroShift ovnk pods running.
+
+For more information on the plugin design and output format, please check the [enhancement proposal](https://github.com/openshift/enhancements/blob/master/enhancements/microshift/microshift-supportability-tools.md).
```

Comment:
I'm not sure we should reference the ER from the docs. It can be confusing because the plugin will evolve and eventually not be like described in the ER.

Reply:
I included it because there is quite some relevant details there. Arent the ER's supposed to evolve with the implementation, if changes are significant enough?

Reply:
It's just, these are user docs and ERs are very technical. A developer should be able to find the ER without the reference.